.github/workflows/build.yml

```diff
@@ -37,21 +37,32 @@ jobs:
       image: rust
     needs: cargo-fmt
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Get latest version of stable rust
         run: rustup update stable
       - name: Install protobuf compiler for the libp2p-core dependency
         uses: arduino/setup-protoc@v1
       - name: Run tests in release
-        run: cargo test --all --release --all-features
+        run: cargo test --all --release --all-features -- --skip ipv6 #  ipv6 tests don't run in github ci https://github.com/actions/runner-images/issues/668  
```

Comment:
On current master branch we have [`return_if_ipv6_is_not_supported`](https://github.com/sigp/discv5/blob/94e09e1fdf30053db7564b29353458c62b009772/src/discv5/test.rs#L152-L168) macro that would be useful to see if IPv6 is supported. You may want to use the macro in the ipv6 tests you added in this PR. 

https://github.com/sigp/discv5/blob/94e09e1fdf30053db7564b29353458c62b009772/src/discv5/test.rs#L352-L370